### ISSUE TITLE
Support for Active Sky 6.5

### ISSUE THREAD

**Issue by UT4UQX** [aloys-c/wx2pfpx#2]
Hi! Your script is great. But can you make some improvements and add support for Active Sky 6.5?
It would be great to see possibility of automatic periodical downloads for TAF and METAR from IVAO or VATSIM servers and data exporting to the main folder of Active Sky.
Is it possible?
Thanks!

**Comment by aloys-c** [comment]
Hi !

So initially the script has nothing to do with Active Sky, it's a weather provider for pfpx and AS is mentioned as its file format (the old one) is used for import of the data.

I don't think I will be modifying the script to provide data for other softwares. Though the script now already provides an option to export the raw data (in excel readable format), so anybody can adapt it to their usage of choice with a bit of data formatting on their side...

**Comment by UT4UQX** [comment]
Thanks for your answer.

But maybe you can add possibility for periodic automatic downloads (every 15 or 20 minutes)? It would be a great addition for functionality.

**Comment by aloys-c** [comment]
The weather data source is updated every 3 hours, the METAR and TAF data is more frequent indeed. Though for pfpx this wouldn't add much as the data is used once at the moment of planning the flight. Why would you need an auto-update ? If it's for a real time update of another software then I don't want to spend some time on it as it's really a pfpx focused script...

**Comment by UT4UQX** [comment]
You're right. It would help to get weather data for another soft (e.g. Active Sky which is now unsupported by HiFi but still best for FS9)
I understand your point of view
Thanks for answers )
